vmm/Cargo.toml

```diff
@@ -28,7 +28,7 @@ event_monitor = { path = "../event_monitor" }
 hypervisor = { path = "../hypervisor" }
 lazy_static = "1.4.0"
 libc = "0.2.94"
-linux-loader = { version = "0.3.0", features = ["elf", "bzimage", "pe"] }
+linux-loader = { git = "https://github.com/jongwu/linux-loader", branch = "uefi", version = "0.3.0", features = ["elf", "bzimage", "pe"] }
```

Comment:
This should eventually point to either somewhere belongs to the CLH community (e.g. a forked branch) or the merged upstream repository (https://github.com/rust-vmm/linux-loader/pull/75)